kolibri/plugins/management/assets/src/vue/data-page/index.vue

```diff
@@ -20,17 +44,31 @@
 
   const Kolibri = require('kolibri');
   module.exports = {
-    methods: {
-      downloadCSV() {
-        window.open(Kolibri.urls['contentsummarylogcsv-list'](), '_blank');
-      },
-    },
     components: {
       'icon-button': require('icon-button'),
     },
+    computed: {
+      summarylogurl() {
+        return Kolibri.urls['contentsummarylogcsv-list']();
+      },
+      sessionlogurl() {
+        return Kolibri.urls['contentsessionlogcsv-list']();
+      },
+    },
   };
 
 </script>
 
 
-<style lang="stylus" scoped></style>
+<style lang="stylus" scoped>
+
+  @require 'jeet'
+
+  @media (min-width: 900px)
```

Comment:
We also have a `$medium-breakpoint` variable which is 619px - any chance this would work? I'm trying to avoid hard-coding values